### PR TITLE
Added JUnit Test for Event.

### PR DESCRIPTION
**Pull request by brandonywl**
Has JUnit test for all classes in Timetable and some Event related Commands

### CODE REVIEW

File: src/main/java/seedu/duke/data/timetable/Event.java [hunk 1]
```diff
@@ -180,10 +180,10 @@ public boolean getRecurring() {
             for (Integer timePeriod : timePeriodsInUnit) {
                 switch (unit) {
                 case REMINDER_DAY:
-                    date = date.plusDays(-timePeriod);
+                    date = this.date.plusDays(-timePeriod);
```
Comment (brandonywl) [inline src/main/java/seedu/duke/data/timetable/Event.java:183]: Was causing a bug with recurring values

**Review by Chongjx — APPROVED** [review]
Looks ok

File: src/test/java/seedu/duke/data/timetable/TimetableTest.java [hunk 2]
```diff
@@ -2,13 +2,100 @@
...
+        timetable.addEvent(dailyEvent);
+        timetable.addEvent(weeklyEvent);
+        timetable.addEvent(monthlyEvent);
+        timetable.addEvent(yearlyEvent);
+        return timetable;
+    }
+
+    /**
+     * Method to initialize a timetable for testing purposes with 1 event.
+     * @return Instantiated Timetable with 1 event.
```
Comment (R-Ramana) [inline src/test/java/seedu/duke/data/timetable/TimetableTest.java:94]: Added Whitespace
```suggestion
     *
     * @return Instantiated Timetable with 1 event.
```

File: src/test/java/seedu/duke/data/timetable/TimetableTest.java [hunk 3]
```diff
@@ -2,13 +2,100 @@
...
     @Test
-    void deleteEvent() {
+    void getReminders_dailyEvent_success() {
+        Timetable timetable = initializeTimetable(dailyEvent);
+        assertEquals(2, timetable.getReminders().size());
+    }
+
+    /**
+     * Method to initialize a timetable for testing purposes with 4 default events.
+     * @return Instantiated Timetable with 4 default events.
```
Comment (R-Ramana) [inline src/test/java/seedu/duke/data/timetable/TimetableTest.java:81]: Added Whitespace
```suggestion
     *
     * @return Instantiated Timetable with 4 default events.
```

File: src/test/java/seedu/duke/data/timetable/EventTest.java [hunk 4]
```diff
@@ -1,5 +1,56 @@
...
+        int i = 0;
+        for (Integer daysBefore : timePeriods) {
+            LocalDate dateTime = TEST_DATE_TIME.minusDays(daysBefore).toLocalDate();
+            assertReminderDate(dateTime, reminderDates.get(i));
+            i++;
+        }
+    }
+
+    /**
+     * Wrapper around assertEquals for code readbility.
```
Comment (R-Ramana) [inline src/test/java/seedu/duke/data/timetable/EventTest.java:49]: Added Whitespace
```suggestion
     * Wrapper around assertEquals for code readbility.
     *
```

**Review by R-Ramana — APPROVED** [review]

**Review by R-Ramana — APPROVED** [review]

**Pull request merged**